vocal/Script_VOCAL_unified.R

```diff
@@ -470,9 +483,8 @@ compute_alert_levels_v1 <- function(pheno_table_wide) {
 var_pheno_summary_wide_with_alert = compute_alert_levels_v1(var_pheno_summary_wide)
 
 prediction_overview = var_pheno_summary_wide_with_alert %>% group_by(alert_level) %>% count()
-message("\n Prediction Results")
-print(prediction_overview)
-message("\n")
+log_info("Prediction Results:")
+log_info(prediction_overview)
```

Comment:
I run with` --verbose`, and the error occurs in this line (487) (error message below), I think because log_info didn't support this data type  
```R
Error in value[[3L]](cond) : `glue` failed in `formatter_glue` on:

 grouped_df [1 × 2] (S3: grouped_df/tbl_df/tbl/data.frame)

Raw error message:

 All unnamed arguments must be length 1

Please consider using another `log_formatter` or `skip_formatter` on strings with curly braces.`glue` failed in `formatter_glue` on:

  $ alert_level: chr "red"

Raw error message:

 All unnamed arguments must be length 1

Please consider using another `log_formatter` or `skip_formatter` on strings with curly braces.`glue` failed in `formatter_glue` on:

  $ n          : int 30

Raw error message:

 All unnamed arguments must be length 1

Please consider using another `log_formatter` or `skip_formatter` on strings with curly braces.`glue` failed in `formatter_glue` on:

  - attr(*, "groups")= tibble [1 × 2] (S3: tbl_df/tbl/data.frame)

Raw error message:

 All unnamed arguments must be length 1

Please consider using another `log_formatter` or `skip_formatter` on strings with curly braces.`glue` failed in `fo
Calls: log_info ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted
```
 I try many different ways to fix it, but I still cannot make it works

for example, convert this list into dataframe
```R
formatter_data_frame <- function(df) {
    pander::pander_return(df, style = "simple")
}
df <- as.data.frame(prediction_overview)
log_info(log_formatter(df))
```

so the current solution in my mind is
```R
if (args$verbose){
print(prediction_overview)
}
```

Reply:
Yes that's totally fine to me, we don't need to do anything fancy here.